syntax = "proto3";

package distribuild;

message EnviromentDesc {
  string compiler_digest = 1;
}